cmd/policy_publish.go

```diff
@@ -32,16 +33,36 @@ import (
 
 func newPolicyPublishCmd() *cobra.Command {
 	var cmd = &cobra.Command{
-		Use:   "publish <orgName>/<policyPackName>",
-		Args:  cmdutil.ExactArgs(1),
+		Use:   "publish [org-name]",
+		Args:  cmdutil.MaximumNArgs(1),
 		Short: "Publish resource policies to the Pulumi service",
-		Long:  "Publish resource policies to the Pulumi service",
+		Long: "Publish resource policies to the Pulumi service\n" +
+			"\n" +
+			"If an organization name is not specified, the current user account is used.",
 		Run: cmdutil.RunFunc(func(cmd *cobra.Command, args []string) error {
+
+			orgName := ""
```

Comment:
Nit: I believe the convention in Go is to use `var orgName string` when you want to set it to the zero value.

Reply:
Yes. I can dig up the link from _Effective Go_ or a style guide somewhere, but you should use `var x Type` instead of `x := Type{}` or `x := 0` if you are trying to just _declare_ a variable rather than _define_ it.